src/utils/LocalStorage.ts

```diff
@@ -0,0 +1,28 @@
+import { Todo } from '../types/Todo';
+
+export const LocalStorage = {
```

Comment:
```suggestion
export const localStorage = {
```
We use PascalCase for components, types, classes only